frontend/src/metabase/actions/utils.ts

```diff
@@ -18,9 +18,6 @@ import Field from "metabase-lib/metadata/Field";
 
 import type { FieldSettings as LocalFieldSettings } from "./types";
 
-export const checkDatabaseSupportsActions = (database: Database) =>
-  database.features.includes("actions");
-
 export const checkDatabaseActionsEnabled = (database: Database) =>
   !!database.settings?.["database-enable-actions"];
```

Comment:
Should we remove `checkDatabaseActionsEnabled` as well?

Reply:
Unfortunately there is one usage with `DataSelector` without `Database` instance that we still need to support